sorting/cycle_sort.cpp

```diff
@@ -1,15 +1,8 @@
 /**
  * @file
- * @brief Implementation of [Cycle
- * sort](https://en.wikipedia.org/wiki/Cycle_sort) algorithm
- *
+ * @brief Implementation of [Cycle sort](https://en.wikipedia.org/wiki/Cycle_sort) algorithm
```

Comment:
I'd like you to put this back as it was (same in lines 5 and 21).

Reply:
They are currently 1-line descriptions.